### PR TITLE
Fix classic theme images width, accessibilty and some lighthouse improvements

### LINKED ISSUE #23352: Classic theme lighthouse and accessibility could be improved
#### Describe the bug

The classic theme could be a bit more improved regarding lighthouse and accessibility improvements

#### Expected behavior

Some aria labels are missing, some images doesn't have any sizes in markup

#### Steps to Reproduce

1. Go to any page on the classic theme
2. Open the chrome console
3. Launch a lighthouse test, that would be enough to see some errors (we can't fix these all)

#### Additional information

* PrestaShop version: develop

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Classic theme lighthouse and accessibility could be improved
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23352.
| How to test?      | See issue

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23353)

### CODE REVIEW

File: themes/classic/templates/_partials/form-fields.tpl [hunk 1]
```diff
@@ -99,7 +99,7 @@
       {elseif $field.type === 'date'}
 
         {block name='form_field_item_date'}
-          <input name="{$field.name}" class="form-control" type="date" value="{$field.value}" placeholder="{if isset($field.availableValues.placeholder)}{$field.availableValues.placeholder}{/if}">
+          <input name="{$field.name}" class="form-control" type="date" value="{$field.value}" placeholder="{if isset($field.availableValues.placeholder)}{$field.availableValues.placeholder}{/if}"{if isset($field.availableValues.placeholder)} aria-label="{$field.availableValues.placeholder}"{/if}>
```
Comment (Progi1984) [inline themes/classic/templates/_partials/form-fields.tpl]:
```suggestion
          <input name="{$field.name}" class="form-control" type="date" value="{$field.value}"{if isset($field.availableValues.placeholder)} placeholder="{$field.availableValues.placeholder}" aria-label="{$field.availableValues.placeholder}"{/if}>
```

File: themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl [hunk 2]
```diff
@@ -67,6 +75,8 @@
               width="100"
```
Comment (Progi1984) [inline themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl]:
```suggestion
```

File: themes/classic/templates/catalog/_partials/product-images-modal.tpl [hunk 3]
```diff
@@ -40,9 +40,11 @@
                 alt="{$product.name}"
               {/if}
               itemprop="image"
+              width="800"
```
Comment (Progi1984) [inline themes/classic/templates/catalog/_partials/product-images-modal.tpl]:
```suggestion
```

File: themes/classic/templates/catalog/_partials/product-images-modal.tpl [hunk 4]
```diff
@@ -40,9 +40,11 @@
...
+              height="800"
```
Comment (Progi1984) [inline themes/classic/templates/catalog/_partials/product-images-modal.tpl]:
```suggestion
              height="{$product.default_image.bySize.large_default.width}"
```

File: themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl [hunk 5]
```diff
@@ -38,12 +38,20 @@
           style="width:100%;"
```
Comment (Progi1984) [inline themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl]:
```suggestion
```

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

**Comment by NeOMakinG** [comment]
@Progi1984 I did my best but some height/widths aren't fine at all to do this

File: themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl (hunk 2 above)
Comment (Progi1984) [reply themes/classic/templates/catalog/_partials/product-cover-thumbnails.tpl]: Ping @NeOMakinG

**Review by Progi1984 — COMMENTED** [review]
A last one and it's good. Good job :tada:

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by atomiix** [comment]
Thanks @NeOMakinG & @sarahdib

**Pull request merged**